### PR TITLE
GEOMESA-821 Fix DataStoreFactorySpi entry in GeoMesa plugin META-INF/…

### PR DESCRIPTION
**Pull request by mpfaffenberger**
…services

### CODE REVIEW

File: geomesa-plugin/src/main/resources/META-INF/services/org.geotools.data.DataStoreFactorySpi [hunk 1]
```diff
@@ -1 +1 @@
-accumulo.data.AccumuloDataStoreFactory
+org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory
```
Comment (elahrvivaz) [inline geomesa-plugin/src/main/resources/META-INF/services/org.geotools.data.DataStoreFactorySpi]: i think this file is actually redundant and can be completely removed, as it comes in already from the geomesa-accumulo module.

**Comment by AndrewAnnex** [comment]
@mpfaffenberger is this PR still needed?

**Comment by mpfaffenberger** [comment]
Yep

**Comment by elahrvivaz** [comment]
@mpfaffenberger See my comment re:removing.

File: geomesa-plugin/src/main/resources/META-INF/services/org.geotools.data.DataStoreFactorySpi (hunk 1 above)
Comment (jnh5y) [inline geomesa-plugin/src/main/resources/META-INF/services/org.geotools.data.DataStoreFactorySpi]: @mpfaffenberger can you try removing the file instead?

**Comment by mpfaffenberger** [comment]
@elahrvivaz @jnh5y updated to remove the file. Not sure what's up with the CLA thing. I definitely signed it and verified that my github email was the same. :octocat:

**Comment by jnh5y** [comment]
https://projects.eclipse.org/user/cla/validate says you're fine Mike, so no worries.

**Comment by anthonyccri** [comment]
Merged.

**Pull request closed without merging**